install.sh

```diff
@@ -48,19 +48,25 @@ if [ "$answer" == "Yes" ] || [ "$answer" == "yes" ] || [ "$answer" == "y" ] || [
   # zsh profile
   if [ -f ~/.zprofile ]; then
     echo "export OPENAI_KEY=$key" >>~/.zprofile
-    echo 'export PATH=$PATH:/usr/local/bin' >>~/.zprofile
+    if [[ ":$PATH:" == *":/usr/local/bin:"* ]]; then
```

Comment:
I think this adds `/usr/local/bin` to $PATH when it already exists in $PATH, and not when it's missing.